paasta_tools/cli/cmds/logs.py

```diff
@@ -559,6 +624,65 @@ def cluster_to_scribe_env(self, cluster):
             return env
 
 
+def generate_start_end_time(time_ago="30m", time=datetime.datetime.utcnow()):
```

Comment:
the default arg for time is a bad idea. Check this out:

```
>>> import datetime
>>> datetime.datetime.utcnow()
datetime.datetime(2016, 6, 6, 13, 26, 37, 870869)
>>> def x(y=datetime.datetime.utcnow()):
...     print y
...
>>> x()
2016-06-06 13:27:06.622029
>>> x()
2016-06-06 13:27:06.622029
>>> x()
2016-06-06 13:27:06.622029
>>> x()
2016-06-06 13:27:06.622029
```

Notice that the value printed is never returned: that's because default args are evaluated once, when the function is defined.

Instead, you should do the following:

```
def generate_start_end_time(time_ago="30m", time=None):
    if time is None:
         time=datetime.datetime.utcnow()
```